template.yml

```diff
@@ -9,16 +9,16 @@ Resources:
     Properties:
       TableName: UsersRoles
       AttributeDefinitions:
-        - AttributeName: EntityId
+        - AttributeName: PK1A
```

Comment:
Generic name for the table's primary key that will allow us to save different entities on the same table.